public/assets/splash-screen.js

```diff
@@ -0,0 +1,11 @@
+var isDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
+var splashScreen = document.getElementById('splash-screen');
+
+// set accent.background-secondary
```

Comment:
@alter-eggo : sorry for the late comment here but do you think we can use `tokens` here now we are loading these in JS? 

Asking as @fbwoolf is cleaning up colours in [this PR](https://github.com/leather-wallet/extension/pull/4964)

`#F5F1ED` relates to `text-primary_dark` but we don't yet have a name for `#2C2A24` and if possible it would be good to have a token for this

Reply:
> do you think we can use tokens here now we are loading these in JS?

hm, don't think so, as we call this script directly in html, and get tokens from the app's library? but I'll be glad to get some advice here

> #F5F1ED relates to text-primary_dark but we don't yet have a name for #2C2A24 and if possible it would be good to have a token for this

hm, not sure here, as it seems we all the same cannot use token...

Reply:
OK no problem, thanks! IIRC the original implementation was in CSS so just double checking now